app/models/recommendation.rb

```diff
@@ -0,0 +1,4 @@
+class Recommendation < ActiveRecord::Base
+  belongs_to :recommendable, polymorphic: true
```

Comment:
I wonder if we need to make this polymorphic now, since we just have TV, or if we can defer some of this later when we have more media? Do you think it'll be tougher to change later on?